SpatialConnect/SCBackendService.m

```diff
@@ -198,7 +201,15 @@ - (RACSignal *)publishReplyTo:(SCMessage *)msg onTopic:(NSString *)topic {
   [session subscribeTopic:msg.replyTo];
   if (session) {
     [session publishData:[msg data] onTopic:topic];
-    return [multicast filter:^BOOL(SCMessage *m) {
+    return [[multicast map:^SCMessage *(RACTuple *t) {
+      NSData *d = (NSData *)[t second];
+      NSError *err;
+      SCMessage *msg = [[SCMessage alloc] initWithData:d error:&err];
+      if (err) {
+        NSLog(@"%@", err.description);
+      }
+      return msg;
+    }] filter:^BOOL(SCMessage *m) {
       if (m.correlationId == msg.correlationId) {
```

Comment:
we should 
1. Subscribe to multicast
2. subscribe to channel
3. publish message
4. return multicast subscription